app/workers/plos_details.rb

```diff
@@ -20,15 +20,33 @@ def perform(plos_paper)
                  "api_key=#{ENV.fetch('PLOS_API_KEY')}"
 
     detail_resp = Net::HTTP.get_response(URI.parse(detail_url))
+    # did we get a proper response?
+    unless detail_resp.is_a? Net::HTTPSuccess
+      # don't bother parsing broken or empty output
+      sleep 6
```

Comment:
Or even better, [configuring the retry delay](https://github.com/mperham/sidekiq/wiki/Error-Handling#configuration):

```ruby
class WorkerWithCustomRetry
  include Sidekiq::Worker
  sidekiq_options :retry => 5

  # The current retry count is yielded. The return value of the block must be 
  # an integer. It is used as the delay, in seconds. 
  sidekiq_retry_in do |count|
    10 * (count + 1) # (i.e. 10, 20, 30, 40)
  end

  def perform(...)
  end
end
```